stdlib/LinearAlgebra/src/uniformscaling.jl

```diff
@@ -122,6 +122,8 @@ isposdef(J::UniformScaling) = isposdef(J.λ)
 (-)(J::UniformScaling, B::BitArray{2})      = J - Array(B)
 (-)(A::AbstractMatrix, J::UniformScaling)   = A + (-J)
 
+(^)(A::UniformScaling, x::Number)           = (A.λ)^x * I
```

Comment:
Is there a reason you prefered this implementation instead of
```suggestion
(^)(A::UniformScaling, x::Number)           = UniformScaling((A.λ)^x)
```
?

Reply:
No, not really, I just assumed any additional overhead would compile away and not matter but for a stdlib, your suggestion does make sense.